user_testing.md

```diff
@@ -0,0 +1,23 @@
+Author: Dan Gunter <dkgunter@lbl.gov>
```

Comment:
I'd put this info below the main heading, so that when rendered it reads more like a regular document.